src/main/java/org/apache/accumulo/access/AeNode.java

```diff
@@ -39,6 +39,21 @@ abstract class AeNode implements Comparable<AeNode> {
 
   abstract int ordinal();
 
+  @Override
+  public boolean equals(Object o) {
+    if (o instanceof AeNode) {
+      return compareTo((AeNode) o) == 0;
+    }
+
+    return false;
```

Comment:
this can be written much more succinctly as:

```java
    return o instanceof AeNode && compareTo((AeNode) o) == 0;
```

Reply:
yeah that is nice